src/pages/Account/Account.scss

```diff
@@ -55,11 +55,21 @@ $page-max-width: 1240px;
   background-color: $dark;
   flex-direction: column;
   flex: 1;
+  padding-bottom: 30px;
```

Comment:
please use vars from `src/scss/constants.scss`

Reply:
fixed it!